### PR TITLE
Ads showing in popular feed

### PR DESCRIPTION
**Pull request by m-hakkestad**
Will add to feed and remove console.logs

### CODE REVIEW

**Comment by codebyjonas** [comment]
Please resolve conflicts @m-hakkestad

**Comment by codebyjonas** [comment]
Please resolve conflicts @m-hakkestad

**Comment by m-hakkestad** [comment]
#158

File: src/pages/Popular.js [hunk 1]
```diff
@@ -11,12 +11,13 @@ class Popular extends React.Component {
         this.state = {
             data: [],
             filters: ['twitter', 'youtube', 'instagram'],
-            limit: 10
+            limit: 100
```
Comment (Hedluund) [inline src/pages/Popular.js]: Change this limit

File: src/pages/Feed.js [hunk 2]
```diff
@@ -33,14 +34,24 @@ class Feed extends React.Component {
...
         let feedContent = null
         if (this.state.data.length > 0) {
-            let filteredContent = this.state.data.filter(content => this.state.filters.includes(content.platform.toLowerCase()))
-            feedContent = filteredContent.map(curContent => {
+            let filteredContent = this.state.data.filter(content => this.filterContent(content))
+            feedContent = filteredContent.map((curContent,index) => {
                 return <FeedComponent
-                    key={curContent.postid}
-                    data={curContent.platformcontent}
+                    key={curContent.postid||index+1000}
```
Comment (Hedluund) [inline src/pages/Feed.js]: See if you can change this to only index

File: src/pages/Popular.js [hunk 3]
```diff
@@ -44,28 +45,36 @@ class Popular extends React.Component {
...
             let filteredContent = this.state.data.filter(content =>
-                this.state.filters.includes(content.platform.toLowerCase())
+                this.filterContent(content)
             )
-            feedContent = filteredContent.map(curContent => {
+            feedContent = filteredContent.map((curContent,index) => {
                 return <PopularComponent
-                    key={curContent.postid}
-                    data={curContent.platformcontent}
+                    key={curContent.postid || index+1000}
```
Comment (Hedluund) [inline src/pages/Popular.js]: Change this to only index as well

**Comment by TandiAB** [comment]
Fullfills req:
FR55

**Pull request merged**